pubsub/streaming-analytics/build.gradle

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'java'
+}
+
+repositories {
+    mavenCentral()
+    maven {
+        url = uri('https://repository.apache.org/content/repositories/snapshots/')
+    }
+
+    maven {
+        url = uri('https://packages.confluent.io/maven/')
+    }
+
+    maven {
+        url = uri('https://repo.maven.apache.org/maven2/')
+    }
+}
+
+dependencies {
+    implementation 'com.github.spotbugs:spotbugs-annotations:4.5.0'
+    implementation 'org.apache.beam:beam-sdks-java-core:2.34.0'
+    implementation 'org.apache.beam:beam-sdks-java-io-google-cloud-platform:2.34.0'
+    implementation 'org.apache.beam:beam-examples-java:2.34.0'
+    implementation 'org.slf4j:slf4j-api:1.7.32'
```

Comment:
Just to make sure, is `log4j` an indirect dependency of any package? If so, do they pull a `log4j` version >= 2.15?

Reply:
- I don't see new releases for `slf4j-api`: https://mvnrepository.com/artifact/org.slf4j/slf4j-api
- Cloud Docs still say it's recommended by Beam: https://cloud.google.com/dataflow/docs/guides/logging#AddingLogging
- To be extra cautious, I will update to log4j 2.16.0